README.md

```diff
@@ -2,43 +2,68 @@
 
 [![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)
 
-## How to create a project with Sunrise data
+## How to create a project with Sunrise Data
 
 Before starting the import, make sure you have access to the [Merchant Center](https://mc.commercetools.com). You will also need to run the application in this repository. Check that you meet the [requirements](#requirements) and [clone](https://help.github.com/articles/cloning-a-repository/) this repository to your computer.
 
 ### 1. Create your project
-1. Open the [Merchant Center](https://mc.commercetools.com) and create an empty project (without sample data).
+
+Open the [Merchant Center](https://mc.commercetools.com) and create an empty project (without sample data).
 
 ### 2. Prepare node environemnt for Import
-1. Go to the root of this project, were the package.json is located.
-2. Install all node dependencies
-    ```js
-        npm install
-    ```
-3. Set the commercetools project credentials as npm config values:
-    <pre>
-        npm config set sunrise:authUrl <i>authUrl</i> - <i>(i.e., auth.commercetools.com)</i>
-        npm config set sunrise:apiUrl <i>apiUrl</i> - <i>(i.e., api.commercetools.com)</i>
-        npm config set sunrise:projectKey <i>projectKey</i>
-        npm config set sunrise:clientId <i>clientId</i>
-        npm config set sunrise:clientSecret <i>clientSecret</i>
-    </pre>
-    * Url variables expect only the host name, not a full URL. 
-
-### 3. Import basic data
-1. import project settings, delivery zones / tax settings, categories and products
-    ```js
+
+Go to the root of this project, where the `package.json` is located and install all node dependencies:
+
+```
+npm install
+```
+      
+### 3. Set your commercetools project credentials
+
+1. In the root of the project create a file named `.env`
+2. Set the `.env` file with your cridentials:
```

Comment:
cridentials -> credentials. 

And please add a hint that when creating an API client in Sunrise you can copy / paste the environment variables as-is as an alternative to dotenv.